pkg/helm/urlfetcher.go

```diff
@@ -124,79 +131,36 @@ func (f *URLFetcher) fetchChart(shaF string) error {
 			return fmt.Errorf("checksum of charts file located at: %s does not match expected SHA file: %s", saved, shaF)
 		}
 	}
-
-	return archiver.Unarchive(saved, f.destDir)
+	targz := archiver.TarGz{Tar: &archiver.Tar{OverwriteExisting: true}}
```

Comment:
somewhere, we should check if the tar is already there (using the sha perhaps). if so, we should not download it repeatedly.